src/directives/ng-file-select.ts

```diff
@@ -25,8 +25,23 @@ export class NgFileSelect {
 
   onFiles(): void {
     let files = this.el.nativeElement.files;
-    if (files.length) {
+    let invalidExtentionFiles: Array<string> = [];
+
+    if (files.length && this.uploader.allowedExtensions.length) {
+      for (var i = 0; i < files.length; i++) {
```

Comment:
files is a FileList object, If we use forEach then it will through following error.

browser_adapter.ts:74 ORIGINAL EXCEPTION: TypeError: files.forEach is not a function